### PR TITLE
Resize video player (fix #11)

### LINKED ISSUE #11: Video and gif display issue.
When viewing gifs and videos on Oldlander, they appear cropped horizontally. It requires the user to scroll sideways to see different parts of the gif/video, but when you do, it's hard not to accidentally open the sidebar, because the videos always require you to swipe left.

Screenshot_20230625_184406_org mozilla fenix

### PR DESCRIPTION
**Pull request by faraplay**
This shrinks the video player so that it fits on small devices, fixing issue #11.
It also expands the width of previews to the width of the whole screen.

Note: I haven't tested this with RES

### CODE REVIEW

**Comment by faraplay** [comment]
I might do a little refactoring of `posts/index.js` while I'm at it

File: js/posts/index.js
Comment (faraplay) [inline js/posts/index.js:1]: What do lines 38-61 do? I think the code in the MutationObserver bit might be buggy but I haven't been able to test it.

Comment (OctoNezd) [reply js/posts/index.js:1]: It waits for RES to load expandos

Comment (faraplay) [reply js/posts/index.js:1]: Can you describe how to get an expando button with the "Expando is not yet ready" title? I've tried loading reddit with RES but I haven't been able to ever get it. Is there some RES setting I need to set?

Comment (OctoNezd) [reply js/posts/index.js:1]: Expando is not ready is (probably) impossible to catch manually, expandos load after page itself loads, so we need to wait for them to load.

**Comment by faraplay** [comment]
Now videos and image previews are shown fullscreen, like how it worked before this pull request with RES enabled.

**Pull request merged**